_data/projects/AMPProject.yml

```diff
@@ -0,0 +1,13 @@
+name: The AMP Project (Accelerated Mobile Pages)
+desc: The AMP Project provides a framework (built on standard web technologies) to make it easy to build fast web pages.
+site: https://github.com/ampproject/amphtml/
+tags:
+- amp
+- css
+- documentation
+- html
+- javascript
+- web
+upforgrabs:
+  name: Great First Issues
+  link: https://github.com/ampproject/amphtml/milestone/25
```

Comment:
> Got it; will it render as a ? in that case?  
Correct, it will render as a `?`, and also (somewhat erroneously) tell users it can't get the issue count because it's not a Github project. The `?!` is for API errors (usu. seen with API rate limiting).

Your decision to put the issues under a milestone isn't unique, so that's totally fine. While I don't necessarily agree with the practice myself, it should link fine. Should/when we move to the Search API (as I mentioned above), then we can make the change to grab the correct count.